libs/wingsdk/src/std/duration.ts

```diff
@@ -29,18 +29,28 @@ export class Duration {
    * @returns a new `Duration` representing `amount` Seconds.
    */
   public static fromSeconds(amount: number): Duration {
+    return new Duration(amount * 1000);
+  }
+
+  /**
+   * Create a Duration representing an amount of milliseconds
+   *
+   * @param amount the amount of milliseconds the `Duration` will represent.
+   * @returns a new `Duration` representing `amount` milliseconds.
+   */
+  public static fromMilliseconds(amount: number): Duration {
     return new Duration(amount);
   }
 
   /**
-   * Return the total number of seconds in this Duration
+   * Return the total number of milliseconds in this Duration
    *
-   * @returns the value of this `Duration` expressed in Seconds.
+   * @returns the value of this `Duration` expressed in Milliseconds.
    */
-  public readonly seconds: number;
+  public readonly milliseconds: number;
```

Comment:
I would consider removing this member and exposing milliseconds similar to how seconds, minutes and hours are exposed. 


Also, should be add days?

Reply:
We can't because we need a base metric to work with (in the private constructor).
That's why seconds was configured in that way when seconds was the base metric.